datadog/resource_datadog_dashboard.go

```diff
@@ -2960,6 +2960,20 @@ func getTimeseriesDefinitionSchema() map[string]*schema.Schema {
 				Schema: getWidgetMarkerSchema(),
 			},
 		},
+		"event": {
+			Type:     schema.TypeList,
+			Optional: true,
+			Elem: &schema.Resource{
+				Schema: getWidgetEventSchema(),
+			},
+		},
+		"yaxis": {
+			Type:     schema.TypeMap,
```

Comment:
Not sure about the type here. If you look at the `HeatmapDefinitionSchema` or `ScatterplotDefinitionSchema` that also have the `yaxis` field, the type used is `TypeList`.

Reply:
@jbenais I fixed up the yaxis to look more like the others, I think. Not sure how you are testing things. Does this work better now?